keyboards/1upkeyboards/sweet16/keymaps/ridingintraffic/config.h

```diff
@@ -0,0 +1,19 @@
+#ifndef CONFIG_RIDINGINTRAFFICTD
+#define CONFIG_RIDINGINTRAFFICTD
+
+/* tap dance stuff*/
+#include "../../config.h"
```

Comment:
This line isn't needed and can be removed. The build script includes all the `config.h` files automatically.